cmd/hubagent/options/options.go

```diff
@@ -158,6 +160,7 @@ func (o *Options) AddFlags(flags *flag.FlagSet) {
 	flags.DurationVar(&o.ForceDeleteWaitTime.Duration, "force-delete-wait-time", 15*time.Minute, "The duration the hub agent waits before force deleting a member cluster.")
 	flags.BoolVar(&o.EnableStagedUpdateRunAPIs, "enable-staged-update-run-apis", true, "If set, the agents will watch for the ClusterStagedUpdateRun APIs.")
 	flags.BoolVar(&o.EnableEvictionAPIs, "enable-eviction-apis", true, "If set, the agents will watch for the Eviction and PlacementDisruptionBudget APIs.")
+	flags.BoolVar(&o.EnableDenyModifyMemberClusterLabels, "enable-deny-modify-member-cluster-labels", false, "If set, users not in the system:masters cannot modify member cluster labels.")
```

Comment:
Nit, but `DenyModifyMemberClusterLabels` / `deny-modify-member-cluster-labels` (remove enable) might be less of a mouthful